SleepEngine/Engine/Utils/stl_utils.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#pragma region Check Find
```

Comment:
why do we need this region?

Reply:
i was trying to group the algos , can remove